docs/config.md

```diff
@@ -44,6 +44,7 @@ for more details.
 | `Npgsql` | [Npgsql](https://www.nuget.org/packages/Npgsql) | ≥6.0.0 | source |
 | `SqlClient` | [Microsoft.Data.SqlClient](https://www.nuget.org/packages/Microsoft.Data.SqlClient) and [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient) | * | source |
 | `StackExchangeRedis` | [StackExchange.Redis](https://www.nuget.org/packages/StackExchange.Redis)  **Not supported on .NET Framework** | ≥2.0.405 < 3.0.0 | source & binary |
+| `Wcf` | [System.ServiceModel.Http](https://www.nuget.org/packages/System.ServiceModel.Http/), [System.ServiceModel.NetTcp](https://www.nuget.org/packages/System.ServiceModel.NetTcp/), [System.ServiceModel](https://www.nuget.org/packages/System.ServiceModel) **No support for server side on .NET Core**. For configuration see [WCF Instrumentation Configuration](wcf-config.md) | * | source |
```

Comment:
We are calling out specific bindings that we are compatible with. There are many others, but I think those are the most popular if I'm remembering correctly. Are there specific reasons that we only support those bindings? It will be good to know if it's a limitation in the instrumentation library, or just with our bootstrapping of the instrumentation library.

Reply:
I am actually not sure we are limited to these 2. I assume the type of communication should not be a limitation for us. [Contrib documentation](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.Wcf/README.md) does not explicitly say only these 2 are supported, but the sample applications use them. I can update the config.md to remove `System.ServiceModel.NetTcp` and `System.ServiceModel.Http` mentions, so there is no confusion.

Reply:
@dszmigielski we should list, in a footnote, the ones that we tested and keep the table comment short as you suggested.

Reply:
I just took a look at the instrumentation library for WCF, and there might be some limitations there. When I worked on some WCF instrumentation in the past I discovered that there were 2 ways to set and get the tracing headers (for context propagation), and it varied depending on whether the binding used a REST or SOAP format. [In one case you could use the headers property like the instrumentation is doing, and in the other case you had to access the header through a http property](https://github.com/newrelic/newrelic-dotnet-agent/blob/efe91f18b9bf8b905847a1b640fbe590730ddf70/src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Wcf3/ChannelFactoryWrapper.cs#L115-L125). I think that the `webHttp` binding is the one where you have to access the http headers through an http property, and then you have to ensure that you don't attempt to access an http property when the nettcp binding is being used (I think there are a couple of other gotchas too).

Reply:
So, yes we may have to call out that not all bindings are fully supported.

Reply:
I have added a note in wcf-config.md. Please ping me if you think it should be moved something else, or rephrased in some way.

Reply:
I think that note should be good enough for now.